src/Reference/Task3.java

```diff
@@ -0,0 +1,72 @@
+package Reference;
+
+import java.util.Scanner;
+
+public class Task3 {
+    public static void main(String[] args) {
+
+        Scanner reader = new Scanner(System.in);
+        System.out.println("Enter a number to determine the range of array (e.g. 2*2, 3*3, ...): ");
```

Comment:
How will it work if a user enters '3*4'?

Reply:
It will not work)

Reply:
Then we have to foresee this and change the implementation